src/utils/evaluation.ts

```diff
@@ -0,0 +1,813 @@
+          metadata: { prompts: initData.capabilities.prompts }
+        });
+      } else {
+        capabilitiesSection.details.push({
+          text: '✗ Server does not support prompts capability',
+          context: 'Prompts help standardize LLM interactions. Add prompts to capabilities.'
+        });
+      }
+
+      // Check for logging support
+      if (initData.capabilities?.logging) {
+        capabilitiesSection.score += 2;
+        capabilitiesSection.details.push({
+          text: '✓ Server supports logging capability',
+          context: 'Logging enables debugging and monitoring of MCP interactions.',
+          metadata: { logging: initData.capabilities.logging }
+        });
+      } else {
+        capabilitiesSection.details.push({
+          text: '✗ Server does not support logging capability',
+          context: 'Logging is useful for debugging MCP integrations. Add logging to capabilities.'
+        });
+      }
+    } else {
+      capabilitiesSection.details.push({
+        text: '✗ Unable to check MCP capabilities',
+        context: 'Server must respond successfully to initialize request to validate capabilities.'
+      });
+    }
+  } catch (error) {
+    capabilitiesSection.details.push({
+      text: `✗ Failed to parse capabilities: ${error}`,
+      context: 'Error parsing server response. Ensure server returns valid JSON-RPC 2.0 format.'
+    });
+  }
+
+  report.sections.capabilities = capabilitiesSection;
+  report.finalScore += capabilitiesSection.score;
+
+  // 3. Transport Layer Modernity (15 points)
+  onProgress('Checking Transport Layer Modernity...');
+  const transportSection: EvaluationSection = {
+    name: 'Transport Layer Modernity',
+    description: 'Evaluates transport methods and protocol support',
+    score: 0,
+    maxScore: 15,
+    details: []
+  };
+
+  // If we connected with the MCP client, we know the transport worked
+  if (mcpClient && connectionUrl) {
+    // Determine transport type from URL
+    const isSSE = connectionUrl.includes('/sse');
+    const isHTTP = connectionUrl.includes('/mcp');
+    
+    if (isHTTP) {
+      transportSection.score += 10;
+      transportSection.details.push({
+        text: '✓ Server supports HTTP streaming (modern standard)',
+        context: 'HTTP streaming (NDJSON) enables real-time bidirectional communication, essential for long-running MCP operations.',
+        metadata: {
+          transportType: 'HTTP/NDJSON',
+          endpoint: connectionUrl,
+          capabilities: 'bidirectional streaming'
+        }
+      });
+    } else if (isSSE) {
+      // SSE gets no points but no penalty
+      transportSection.details.push({
+        text: '⚠ Server supports SSE (legacy standard - no points awarded)',
+        context: 'Server-Sent Events (SSE) is an older unidirectional streaming method. Modern MCP servers should use NDJSON streaming instead.'
+      });
+    }
```

Comment:
It should be a failed check if it's neither Http streaming nor SSE, so add a general else block as well